serverless/functions/join-stream-as-speaker.js

```diff
@@ -129,24 +145,6 @@ module.exports.handler = async (context, event, callback) => {
     return callback(null, response);
   }
 
-  // Lower the participant's hand
-  try {
```

Comment:
I think it is simplest to modify these user sync maps from the webhooks for consistency. It's easier to keep things straight. And it's arguably better UX. Updates are made to the viewers and speakers maps at the same time. And so we minimize any gap that the UI would have moving a viewer to speaker. 

I can see an argument that it doesn't completely match reality but that seems minor. It may make sense to reconsider when we the player streamer callback adds a disconnect event.